File: Python/Algorithm/programmers/findKim.py
# [findKim.py]
# 서울에서 김서방 찾기

# First
def solution1(seoul):
    return "김서방은 "+str(seoul.index("Kim"))+"에 있다"

# Second
def solution2(seoul):
    return "김서방은 {}에 있다".format(seoul.index("Kim"))

# Third
def solution3(seoul):
    return ("김서방은 %d에 있다" %seoul.index("Kim"))

File: Java/Algorithm/programmers/findKim.java
// [findKim.java]
// 서울에서 김서방 찾기

class Solution {
  public String solution(String[] seoul) {
      String answer = "";

      // 1. 전체에서 Kim을 찾으면 종료
      for (int i=0; i<seoul.length; i++) {
          if (seoul[i].equals("Kim")) {
              answer = "김서방은 "+ i + "에 있다";
              break;
          }
      }
      // 2. Kim의 위치 출력
      return answer;
  }
}